channels/es.m3u

```diff
@@ -1,11 +1,11 @@
 #EXTM3U x-tvg-url="http://195.154.221.171/epg/guidespain.xml.gz"
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://i.imgur.com/SvUWRQm.png" group-title="News",24 Horas
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://i.imgur.com/SvUWRQm.png" group-title="News",24 Horas HD
```

Comment:
I don't have this channel in HD quality at all:
<img width="406" alt="Captura de pantalla 2020-04-29 a las 14 56 13" src="https://user-images.githubusercontent.com/7253922/80593419-ddefe480-8a29-11ea-837c-44a9cb320150.png">